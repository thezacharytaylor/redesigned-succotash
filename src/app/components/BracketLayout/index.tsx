import PropTypes from 'prop-types';
import {
  ArrowPathRoundedSquareIcon,
  ArrowTurnRightUpIcon,
  ArrowTurnRightDownIcon,
} from '@heroicons/react/24/solid';

interface BracketProps {
  players: string[];
}

const BracketLayout: React.FC<BracketProps> = ({ players }) => {
  const rows: string[][] = [];
  for (let i = 0; i < players.length; i += 2) {
    rows.push([players[i], players[i + 1]]);
  }

  return (
    <>
      <table className="table bg-base-300">
        <tbody>
          {rows.map(([player1, player2]) => (
            <tr key={player1 + player2}>
              <td className="p-0.5 bg-base-300">
                <div className="bracket-inner-wrap w-full py-1 px-2 bg-base-200">
                  {player1}
                  <br />
                  <ArrowPathRoundedSquareIcon className="w-5 h-5 mx-auto" />
                  {player2}
                </div>
              </td>
            </tr>
          ))}
        </tbody>
      </table>
    </>
  );
};

export default BracketLayout;
