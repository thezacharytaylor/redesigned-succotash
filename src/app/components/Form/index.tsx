import PropTypes from 'prop-types';
import Input from './Input';
import { CSSTransition } from 'react-transition-group';

const Form = ({ submit, input, info, golfers }) => {
  // const [valueState, setValueState] = useState(info.name);

  const handleUpdate = (event, golferName) => {
    event.preventDefault();
    // nameField = golfer.name;
    // console.log(nameField);
    // if (nameField.current !== undefined) {
    //   // nameField.current = golferName;
    //   console.log(nameField.current);
    // }
    // return <p className="absolute px-2 bg-white">Hi yea</p>;
  };

  return (
    <>
      <form action="#" onSubmit={submit} className="relative flex">
        <Input name="name" placeholder="Name" value={info.name} func={input} />
        <div className="absolute bottom-0 w-full">
          <CSSTransition
            in={info.name.length > 0 && golfers.length > 0}
            timeout={300}
            classNames="slide-vertical"
            unmountOnExit
          >
            <ul className={`absolute bg-white`}>
              {golfers.map((golfer, index) => {
                return (
                  <li key={index}>
                    <button
                      className="w-full p-2 text-left bg-transparent border-0 hover:bg-gray-300"
                      onClick={event => handleUpdate(event, golfer.name)}
                      name="name"
                      value={golfer.name}
                    >
                      {golfer.name}
                    </button>
                  </li>
                );
              })}
            </ul>
          </CSSTransition>
        </div>
        <Input name="score" placeholder={1} value={info.score} func={input} />
        <button
          type="submit"
          className="px-4 text-lg text-white bg-gray-700 lg:text-2xl"
        >
          +
        </button>
      </form>
    </>
  );
};

Form.Props = {
  submit: PropTypes.func,
  input: PropTypes.func,
  info: PropTypes.shape({
    name: PropTypes.string,
    score: PropTypes.number,
  }),
};

export default Form;
