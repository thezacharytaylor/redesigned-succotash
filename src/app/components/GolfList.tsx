import React from 'react';

function Golfers() {
  const [inputGolfer, setInputGolfer] = React.useState('');
  const [golfers, setGolfers] = React.useState([
    {
      name: 'Ken Price',
      score: 20,
      date: '10/24/2022',
      inCup: true,
      qualified: true,
      checkedIn: true,
      seed: 1,
    },
    {
      name: 'David Green',
      score: 20,
      date: '10/24/2022',
      inCup: true,
      qualified: true,
      checkedIn: true,
      seed: 2,
    },
  ]);

  const handleGolferInput = event => {
    setInputGolfer(event.target.value);
  };

  const addGolfer = () => {};

  return (
    <div className="max-w-md p-10 my-5 bg-gray-200 border-solid rounded-md shadow-sm border-slate-50">
      Golfers
    </div>
    <form action="#" onSubmit={addTodo}>
        <input
          type="text"
          className="border-gray-500"
          value={todoInput}
          onChange={handleTodoInput}
        />
        <button type="submit" className="px-2 text-white bg-gray-700">
          +
        </button>
      </form>
  );
}

export default Golfers;
