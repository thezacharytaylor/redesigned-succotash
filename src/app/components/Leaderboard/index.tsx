import YearDisplay from '../YearDisplay';
import PropTypes from 'prop-types';
import { useEffect, useRef } from 'react';
import Row from './Row';

const defaultRefArray: (HTMLButtonElement | null)[] = [];

const Leaderboard = ({ headings, players }) => {
  const buttonRefs = useRef(defaultRefArray);
  const rows = ['index', 'name', 'score', 'date'];

  const buttonRefLoad = button => {
    buttonRefs.current.push(button);
  };

  const insertCutOff = index => {
    console.log(index === 15);

    return (
      <>
        {index === 16 && (
          <tr key="cutoff" className="bg-red-700">
            <td className="bg-red-700"></td>
            <td className="bg-red-700">
              <div className="p-0 bg-transparent border-none cursor-default">
                Cut Off
              </div>
            </td>
            <td className="bg-red-700"></td>
            <td className="bg-red-700">Cut Off</td>
          </tr>
        )}
      </>
    );
  };

  const handleKeyUp = event => {
    //
  };

  return (
    <>
      <h2
        className="mb-4 text-lg font-bold text-gray-800"
        id="leaderboard-header"
      >
        <YearDisplay> Individual Qualifying High Scores</YearDisplay>
      </h2>
      <div className="overflow-x-auto">
        <table
          className="table w-full table-zebra"
          aria-labelledby="leaderboard-header"
          role="grid"
        >
          <thead>
            <tr>
              {headings.map((heading, index) => (
                <th
                  key={index}
                  className={index === 1 ? 'text-left' : ''}
                  scope="col"
                >
                  {heading}
                </th>
              ))}
            </tr>
          </thead>
          <tbody>
            {players
              .sort((a, b) => b.score - a.score)
              .map((player, index) => {
                return (
                  <>
                    {insertCutOff(index)}
                    <Row
                      key={index}
                      index={index}
                      rows={rows}
                      buttonRefFunc={buttonRefLoad}
                      player={player}
                    />
                  </>
                );
              })}
          </tbody>
        </table>
      </div>
    </>
  );
};

Leaderboard.Props = {
  headings: PropTypes.array,
  players: PropTypes.array,
};

export default Leaderboard;
