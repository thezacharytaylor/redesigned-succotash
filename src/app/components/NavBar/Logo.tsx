import * as React from 'react';
import styled from 'styled-components/macro';

export function Logo() {
  return (
    <Wrapper>
      <img
        src="/logo192.png"
        alt="logo"
        width="25"
        height="25"
        className="mr-1.5 pb-2"
      />
      <Title>Cooking: KineticCup</Title>
      <Description></Description>
    </Wrapper>
  );
}

const Wrapper = styled.div`
  display: flex;
  align-items: center;
`;

const Title = styled.div`
  font-size: 1.25rem;
  color: ${p => p.theme.text};
  font-weight: bold;
  margin-right: 1rem;
`;

const Description = styled.div`
  font-size: 0.875rem;
  color: ${p => p.theme.textSecondary};
  font-weight: normal;
`;
