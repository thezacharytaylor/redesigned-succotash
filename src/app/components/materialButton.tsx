import React from 'react';

function materialButton() {
  return (
    <div>
      <button className="rounded-full p-3 bg-lime-600 hover:bg-lime-500 ease-in-out text-white">
        Click me!
      </button>
    </div>
  );
}

export default materialButton;
