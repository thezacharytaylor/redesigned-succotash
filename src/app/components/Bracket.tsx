import { useEffect, useState } from 'react';
import { useDispatch, useSelector } from 'react-redux';
import { setName, setScore } from 'features/player-input/inputSlice';
import { RootState } from 'store/reducers';
import Form from './Form';
import dayjs from 'dayjs';
import BracketLayout from './BracketLayout';
import PlayerData from '../../store/golfers.json';
import DefaultPlayer from 'app/interfaces/defaultPlayer';

// TODO: Monitor if all quarters/semis/finals are set, to for a11y nav? or just make an a11y nav?
// TODO: Add a11y nav for bracket
// TODO: Style brackets
// TODO: feed one sheet here/monitor state to appropriately display brackets
// TODO: Show brackets & show scores of brackets

const defaultPlayer: DefaultPlayer = {
  id: -1,
  name: '',
  score: 0,
  date: '',
  inCup: false,
  qualified: false,
  checkedIn: false,
  seed: 16,
  previousPlayer: false,
};

const Players = () => {
  const dispatch = useDispatch();
  const [players, setPlayers] = useState<DefaultPlayer[]>([defaultPlayer]);

  useEffect((): void => {
    setPlayers([...PlayerData]);
  }, []);
};

const Bracket = () => {
  return (
    <div className="my-5 border-solid rounded-md shadow-sm bg-primary-content border-slate-50">
      <div className="overflow-x-auto flex">
        <BracketLayout
          players={[
            'Jon Smith',
            'Hank Hill',
            'Chris Crill',
            'Ken Price',
            'Jeremy Morris',
            "Tom O'Conner",
            'Mike Smith',
            "Antonio T'Von",
          ]}
        />
        <BracketLayout
          players={['Jon Smith', 'Ken Price', 'Jeremy Morris', 'Mike Smith']}
        />
        <BracketLayout players={['Ken Price', 'Mike Smith']} />
        <BracketLayout players={['Ken Price', 'Jeremy Morris']} />
        <BracketLayout players={['Jon Smith', 'Jeremy Morris']} />
        <BracketLayout
          players={['Jon Smith', 'Ken Price', 'Jeremy Morris', 'Mike Smith']}
        />
        <BracketLayout
          players={[
            'Jon Smith',
            'Hank Hill',
            'Chris Crill',
            'Ken Price',
            'Jeremy Morris',
            "Tom O'Conner",
            'Mike Smith',
            "Antonio T'Von",
          ]}
        />
      </div>
    </div>
  );
};

export default Bracket;
