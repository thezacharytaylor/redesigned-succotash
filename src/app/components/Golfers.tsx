import { useEffect, useState } from 'react';
// import { getEvents } from 'features/calendar/calendarSlice';
import { useDispatch, useSelector } from 'react-redux';
import { setName, setScore } from 'features/golfer-input/inputSlice';
import { RootState } from 'store/reducers';
import LeaderBoard from './Leaderboard';
import Form from './Form';
import dayjs from 'dayjs';
import GolferData from '../../store/golfers.json';
import YearDisplay from './YearDisplay';

const initialInput = {
  name: '',
  score: 1,
};

const defaultGolfer = {
  id: -1,
  name: '',
  score: 0,
  date: '',
  inCup: false,
  qualified: false,
  checkedIn: false,
  seed: 16,
  previousPlayer: false,
};

const Golfers = () => {
  const input = useSelector(state => state.golferInput);
  const disptach = useDispatch();
  const [golferInput, setGolferInput] = useState(initialInput);
  const [golfers, setGolfers] = useState([defaultGolfer]);
  const [filteredGolfers, setFilteredGolfers] = useState([defaultGolfer]);

  useEffect(() => {
    setGolfers([...GolferData]);
  }, []);

  const handleGolferInput = (event: {
    target: { name: string; value: string };
  }) => {
    const { name, value } = event.target;
    setGolferInput({ ...golferInput, [name]: value });

    if (name === 'name') {
      handlePrediction(value);
    }
  };

  const handlePrediction = value => {
    if (value.length > 0) {
      const newGolfers = golfers.filter(golfer => {
        const uniformName: string = golfer.name.toUpperCase();
        const uniformValue: string = value.toUpperCase();
        return uniformName.startsWith(uniformValue) ? golfer.name : '';
      });

      setFilteredGolfers([...newGolfers]);
    }
  };

  const handleFormSubmit = (event: { preventDefault: () => void }) => {
    event.preventDefault();
    const updatedGolfer = updateGolfer();

    if (!updatedGolfer) {
      addGolfer();
    }
  };

  const updateGolfer = () => {
    let locatedGolfer = false;
    const updatedGolfers = golfers.map(golfer => {
      if (golfer.name === golferInput.name) {
        golfer.score = golferInput.score;
        golfer.date = dayjs().format('MM/DD/YYYY');
        locatedGolfer = true;
      }

      return golfer;
    });

    setGolfers(updatedGolfers);
    return locatedGolfer;
  };

  // This might need to be a custom hook
  const addGolfer = () => {
    setGolfers([
      ...golfers,
      {
        id: golfers[golfers.length - 1].id + 1,
        name: golferInput.name,
        score: golferInput.score,
        date: dayjs().format('MM/DD/YYYY'),
        inCup: true,
        qualified: true,
        checkedIn: true,
        seed: 2,
        previousPlayer: false,
      },
    ]);

    setGolferInput({ name: '', score: 1 });
  };

  // const deleteGolfer = id => {};

  return (
    <div className="p-10 my-5 bg-gray-200 border-solid rounded-md shadow-sm border-slate-50">
      <h2 className="mb-4 text-lg font-bold text-gray-800">
        <YearDisplay> Individual Qualifying High Scores</YearDisplay>
      </h2>
      <LeaderBoard
        headings={['Rank', 'Name', 'Score', 'Date']}
        players={golfers}
      />
      <hr className="my-4 border-gray-800 border-solid" />
      <h2 className="mb-4 text-lg font-bold text-gray-800">Add Score</h2>
      <Form
        submit={handleFormSubmit}
        input={handleGolferInput}
        info={golferInput}
        golfers={filteredGolfers}
      />
    </div>
  );
};

export default Golfers;
