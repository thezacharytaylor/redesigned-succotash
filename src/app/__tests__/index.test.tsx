import React from 'react';
import { render, screen } from '@testing-library/react';

import { NavBar } from 'app/components/NavBar';
import { App } from '../index';
import { Provider } from 'react-redux';
import { HelmetProvider } from 'react-helmet-async';
import { configureAppStore } from 'store/configureStore';

const store = configureAppStore();

test('should render and match the snapshot', () => {
  render(
    <Provider store={store}>
      <HelmetProvider>
        <React.StrictMode>
          <App />
        </React.StrictMode>
      </HelmetProvider>
    </Provider>,
  );
  const headerElement = screen.getByText('Kinetic Cup');
  expect(headerElement).toBeInTheDocument();
});
