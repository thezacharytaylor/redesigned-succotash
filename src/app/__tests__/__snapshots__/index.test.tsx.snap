// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render and match header text 1`] = `
Array [
  .c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c4 {
  font-size: 1.25rem;
  font-weight: bold;
  margin-right: 1rem;
}

.c5 {
  font-size: 0.875rem;
  font-weight: normal;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-right: -1rem;
}

.c7 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 0.25rem 1rem;
  font-size: 0.875rem;
  font-weight: 500;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c7:hover {
  opacity: 0.8;
}

.c7:active {
  opacity: 0.4;
}

.c7 .icon {
  margin-right: 0.25rem;
}

.c2 {
  width: 960px;
  margin: 0 auto;
  padding: 0 1.5rem;
  box-sizing: content-box;
}

.c0 {
  box-shadow: 0 1px 0 0 #00141b;
  height: 4rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: fixed;
  top: 0;
  width: 100%;
  background-color: #00141b;
  z-index: 2;
}

.c0 .c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@supports (-webkit-backdrop-filter:blur(10px)) or (backdrop-filter:blur(10px)) {
  .c0 {
    -webkit-backdrop-filter: blur(10px);
    backdrop-filter: blur(10px);
    background-color: #00141b;
  }
}

<header
    className="c0"
  >
    <div
      className="c1 c2"
    >
      <div
        className="c3"
      >
        <div
          className="c4"
        >
          Kinetic Cup
        </div>
        <div
          className="c5"
        >
          Fore!
        </div>
      </div>
      <nav
        className="c6"
      >
        <a
          className="c7"
          href="https://cansahin.gitbook.io/react-boilerplate-cra-template/"
          rel="noopener noreferrer"
          target="_blank"
          title="Documentation Page"
        >
          <svg
            aria-hidden="true"
            className="w-5 h-5 mr-1"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M4.125 3C3.089 3 2.25 3.84 2.25 4.875V18a3 3 0 003 3h15a3 3 0 01-3-3V4.875C17.25 3.839 16.41 3 15.375 3H4.125zM12 9.75a.75.75 0 000 1.5h1.5a.75.75 0 000-1.5H12zm-.75-2.25a.75.75 0 01.75-.75h1.5a.75.75 0 010 1.5H12a.75.75 0 01-.75-.75zM6 12.75a.75.75 0 000 1.5h7.5a.75.75 0 000-1.5H6zm-.75 3.75a.75.75 0 01.75-.75h7.5a.75.75 0 010 1.5H6a.75.75 0 01-.75-.75zM6 6.75a.75.75 0 00-.75.75v3c0 .414.336.75.75.75h3a.75.75 0 00.75-.75v-3A.75.75 0 009 6.75H6z"
              fillRule="evenodd"
            />
            <path
              d="M18.75 6.75h1.875c.621 0 1.125.504 1.125 1.125V18a1.5 1.5 0 01-3 0V6.75z"
            />
          </svg>
          Documentation
        </a>
        <a
          className="c7"
          href="https://github.com/react-boilerplate/react-boilerplate-cra-template"
          rel="noopener noreferrer"
          target="_blank"
          title="Github Page"
        >
          <svg>
            github-icon.svg
          </svg>
          Github
        </a>
        <a
          className="c7"
          href="/leaderboard"
          target=""
          title="Leaderboard"
        >
          <svg
            aria-hidden="true"
            className="w-5 h-5 mr-1"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M18.375 2.25c-1.035 0-1.875.84-1.875 1.875v15.75c0 1.035.84 1.875 1.875 1.875h.75c1.035 0 1.875-.84 1.875-1.875V4.125c0-1.036-.84-1.875-1.875-1.875h-.75zM9.75 8.625c0-1.036.84-1.875 1.875-1.875h.75c1.036 0 1.875.84 1.875 1.875v11.25c0 1.035-.84 1.875-1.875 1.875h-.75a1.875 1.875 0 01-1.875-1.875V8.625zM3 13.125c0-1.036.84-1.875 1.875-1.875h.75c1.036 0 1.875.84 1.875 1.875v6.75c0 1.035-.84 1.875-1.875 1.875h-.75A1.875 1.875 0 013 19.875v-6.75z"
            />
          </svg>
          Leaderboard
        </a>
      </nav>
    </div>
  </header>,
  .c1 {
  width: 960px;
  margin: 0 auto;
  padding: 0 1.5rem;
  box-sizing: content-box;
}

.c2 .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@supports (-webkit-backdrop-filter:blur(10px)) or (backdrop-filter:blur(10px)) {

}

<div
    className="c0 c1"
  >
    <div
      className="container px-4 mx-auto"
    >
      <h1
        className="pb-4 underlin"
        style={
          Object {
            "color": "white",
          }
        }
      >
        Leaderboard
      </h1>
      <div
        className="p-10 my-5 bg-gray-200 border-solid rounded-md shadow-sm border-slate-50"
      >
        <h2
          className="mb-4 text-lg font-bold text-gray-800"
          id="leaderboard-header"
        >
            Individual Qualifying High Scores
        </h2>
        <div
          className="overflow-x-auto"
        >
          <table
            aria-labelledby="leaderboard-header"
            className="table w-full table-zebra"
            role="grid"
          >
            <thead>
              <tr>
                <th
                  className=" !bg-green-900 text-white"
                  scope="col"
                >
                  Rank
                </th>
                <th
                  className="text-left !bg-green-900 text-white"
                  scope="col"
                >
                  Name
                </th>
                <th
                  className=" !bg-green-900 text-white"
                  scope="col"
                >
                  Score
                </th>
                <th
                  className=" !bg-green-900 text-white"
                  scope="col"
                >
                  Date
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td
                  className="pl-6 !bg-green-700"
                >
                  <button
                    className="p-0 text-white bg-transparent border-none cursor-default"
                  >
                    1
                  </button>
                </td>
                <td
                  className=" !bg-green-700"
                >
                  <button
                    className="p-0 text-white bg-transparent border-none cursor-default"
                  />
                </td>
                <td
                  className=" !bg-green-700"
                >
                  <button
                    className="p-0 text-white bg-transparent border-none cursor-default"
                  >
                    0
                  </button>
                </td>
                <td
                  className=" !bg-green-700"
                >
                  <button
                    className="p-0 text-white bg-transparent border-none cursor-default"
                  />
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <hr
          className="my-4 border-gray-800 border-solid"
        />
        <h2
          className="mb-4 text-lg font-bold text-gray-800"
        >
          Add Score
        </h2>
        <form
          action="#"
          className="relative flex input-group"
          onSubmit={[Function]}
        >
          <input
            className="input input-bordered w-full max-w-xs false"
            name="name"
            onChange={[Function]}
            placeholder="Name"
            type="text"
            value=""
          />
          <div
            className="absolute bottom-0 w-full"
          />
          <input
            className="input input-bordered w-full max-w-xs max-w-[4rem]"
            name="score"
            onChange={[Function]}
            placeholder={1}
            type="text"
            value={0}
          />
          <button
            className="px-4 text-lg text-white bg-gray-700 lg:text-2xl btn"
            type="submit"
          >
            +
          </button>
        </form>
      </div>
    </div>
  </div>,
]
`;
